hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/update/server/MockCRLStore.java

```diff
@@ -106,7 +106,7 @@ public BigInteger issueCert() throws Exception {
     List<CRLInfo> crlInfos =
         scmCertStore.getCrls(ImmutableList.of(crlId.get()));
 
-    if (crlInfos.isEmpty()) {
+    if (crlInfos == null || crlInfos.isEmpty()) {
```

Comment:
When `crlInfos ` can be `null`? 
The value is fetched from. ``scmCertStore.getCrls(ImmutableList.of(crlId.get()));`` and this doesn't look like returning a `null` in any case. It is in general returning an empty list, if it can not find anything?

Secondly, If I understand the code correct, now if `crlInfos` is null, we will log the line below, so, will this ` crlInfos.get(0)` not throw a NPE since `crlInfos` is null?

Thirdly, How this `null` check, preventing an `IndexOutOfBoundsException`

Reply:
When `crlInfos` is empty, `crlInfos.get(0)` will have something feels unreasonable. After I communicated with Xiaoyu Yao, it was because of the original negligence, it should be `if (!crlInfos.isEmpty())`.